.github/workflows/ci.yml

```diff
@@ -213,7 +213,20 @@ jobs:
 
       - name: Cargo (package-level)
         if: ${{ matrix.cargo_command && matrix.package }}
-        run: cargo ${{ matrix.cargo_command }} -p ${{ matrix.package }} ${{ matrix.args }}
+        run: |
+         if [[ ${{ matrix.package }} == "fuels" ]]; then
+           if [[ ${{ matrix.cargo_command }} == "test" ]]; then
+             cargo test -p fuels --all-targets --features "default fuel-core-lib test-type-paths" --workspace
+           fi
+           if [[ ${{ matrix.cargo_command }} == "clippy" ]]; then
+             cargo clippy -p fuels --all-targets --features "default fuel-core-lib test-type-paths"
+           fi
+           if [[ ${{ matrix.cargo_command }} == "check" ]]; then
```

Comment:
Should these be changed to use `elif` instead? I believe all `if` expressions are evaluated; however, when using `elif` if one condition is met then the rest will not be evaluated which saves a bit of computation

Reply:
why not :) Ill do it